### ISSUE TITLE
OneToMany bidirectional  relationship Generate Error

### ISSUE THREAD

**Issue by ouyangshixiong** [jhipster/jhipster-uml#166]
### The issue
Two entity A and B with bidirectional relationship OneToMany.
 It should generate two entity with A contains OneToMany annotation,B contains ManyToOne annotation.
But new version  of jhipster-uml will generate two OneToMany annotation.
see the picture attached.
---
#### The stacktrace

```
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1574) ~[spring-beans-4.2.4.RELEASE.jar:4.2.4.RELEASE]
	... 102 common frames omitted
Caused by: org.hibernate.AnnotationException: mappedBy reference an unknown target entity property: com.youai.test2.domain.YaUser.yaOrder in com.youai.test2.domain.YaOrder.yaUsers
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:769) ~[hibernate-core-4.3.11.Final.jar:4.3.11.Final]
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:729) ~[hibernate-core-4.3.11.Final.jar:4.3.11.Final]
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:70) ~[hibernate-core-4.3.11.Final.jar:4.3.11.Final]
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1697) ~[hibernate-core-4.3.11.Final.jar:4.3.11.Final]
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1426) ~[hibernate-core-4.3.11.Final.jar:4.3.11.Final]
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1846) ~[hibernate-core-4.3.11.Final.jar:4.3.11.Final]
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl$4.perform(EntityManagerFactoryBuilderImpl.java:857) ~[hibernate-entitymanager-4.3.11.Final.jar:4.3.11.Final]
	... 110 common frames omitted

2016-03-20 09:02:41.706 ERROR 2412 --- [ost-startStop-1] o.apache.catalina.core.StandardContext   : One or more Filters failed to start. Full details will be found in the appropriate container log file
```

---

### Steps to reproduce
![problem](https://cloud.githubusercontent.com/assets/2086862/13902250/ff550896-ee7c-11e5-8b64-59f59fc19142.png)

---

### The XMI or JDL file

relationship OneToMany{
	/**
	 *  用户和订单一对多关系
	 *
	 *
	*/
	YaUser{YaOrder} to YaOrder{YaUser}
}

/**
 *  用户信息
 *
 *
*/
entity YaUser{
	name				String 		required	minlength(1)		maxlength(200),
	mobileNumber		String,
	facebook			String,
	facebookToken       String,
	email				String,
	password			String		required,
	hasPrivatePot		Boolean,
	authToken			String,
	authTokenUpdateTime LocalDate,
	smUsedCount			Integer,
	smLastCheckDate		LocalDate,
	smLastContent		String,
	smLastValidateCode  String,
	smExpireTime2		Long
}

/**
 *  订单
 *
 *
*/
entity YaOrder{
	orderid 			String		required,
	useCoupon			Boolean,
	orderDate			LocalDate,
	productAmount		Integer,
	orderTotalPrice		Float		required,
	orderStatusChangeTime Long,
	deliveryDate		LocalDate,
	deliveryPeriod		String,
	deliveryManName		String		        	minlength(2)		maxlength(20),
	deliveryPhone		String,
	hasPrivatePot		Boolean,
	usePrivatePot		Boolean,
	parentOrderid		String,		
	isParentOrder       Boolean     required,
	orderRealPrice		Float		required
}

---

### Versions
latest version

| JHipster           | JHipster UML           |
| ------------------ | ---------------------- |
| `JHipster Version` | `JHipster UML version` |

**Comment by MathieuAA** [comment]
Hi, this issue has been fixed and has already been reported here #162.
It hasn't been released yet however.

**Comment by ouyangshixiong** [comment]
OK. Thank you @MathieuAA
